packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -95,8 +95,8 @@ class BpkDataTable<Row> extends Component<Props<Row>, State<Row>> {
       return;
     }
 
-    // See: https://reactjs.org/docs/events.html#event-pooling
-    const eventTarget = event.currentTarget;
+    // See: https://reactjs.org/docs/legacy-event-pooling.html
+    const eventTarget = event.target;
```

Comment:
It looks like I introduced this bug FWIW. See https://github.com/Skyscanner/backpack/pull/2064/files#r557349473. I think what we should do is my suggestion from there i.e. checking the type to ensure it's `Element`(which it will always be)